### ISSUE TITLE
Displaying BED Files as Separate Tracks

### ISSUE THREAD

**Issue by mllamasl** [adamewing/methylartist#93]
It would be a great option to extend the current BED file highlighting functionality `--highlight_bed` to allow BED files to be displayed as separate tracks, as for example if you want to see CpG islands. This feature would be very useful for users to visualize regions of interest alongside methylation data in distinct panels.

Thanks for this amazing tool!
María

**Comment by adamewing** [comment]
Thank you for using it! Do you mean an option to view multiple .bed files such that they're each shown in their own track?

**Comment by mllamasl** [comment]
Yes, exactly! To clarify further, I've attached a figure that ilustrates what I have in mind. 

![testing_methylartist+bed_track](https://github.com/user-attachments/assets/e543fc33-0b8a-45b9-a9fb-d9da76073423)

Each `.bed` file would be shown in its own track, similar to how CpG islands are shown in this figure. Another example that came to my mind could be a `bed` file with repetitive elements, allowing users to easily spot regions with different methylation patterns.

Thanks again for your quick response,
María

**Comment by silvia-s** [comment]
Great idea! I'd like to use that feature too, when available
